src/widgets/canvas/mod.rs

```diff
@@ -111,26 +111,28 @@ impl Grid for BrailleGrid {
 }
 
 #[derive(Debug, Clone)]
-struct DotGrid {
+struct SingleCellGrid {
```

Comment:
I would rather have it called `CharGrid`. WDYT ?

Reply:
Yeah, I also like it